File: Fundamentals with JS/T10 - Functions/exercise/02. Add and Subtract.js
function addAndSubtract(a, b, c) {
    function sum(a, b) {
        return a + b;
    }
    let sumResult = sum(a, b);
    let subtract = (sumResult, c) => sumResult - c;
    let subtractResult = subtract(sumResult, c);

    console.log(subtractResult);
}
addAndSubtract(23, 6, 10);
addAndSubtract(1, 17, 30);
addAndSubtract(42, 58, 100);

File: Fundamentals with Python/T09 - Functions/exercise/02. Add and Subtract.py
a = int(input())
b = int(input())
c = int(input())

def add_and_subtract(a, b, c):

    def sum_numbers(a, b):
        result = a + b
        return result
    def subtract(sum, c):
        result = sum - c
        return result

    return subtract(sum_numbers(a, b), c)

print(add_and_subtract(a, b, c))